template.yaml

```diff
@@ -39,6 +39,10 @@ Parameters:
     Description: |
       The export name suffix for the function ARN
     Default: LogIngestionFunctionArn
+  PermissionsBoundary:
+    Type: String
+    Description: IAM Role PermissionsBoundary (optional)
+    Default: !Ref AWS::NoValue
```

Comment:
Is this correct way to provide optional parameter? PermissionsBoundary is string parameter with required: no

https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html#sam-function-permissionsboundary

Reply:
I believe you can just leave the `Default` line out.

Reply:
Is there not an error setting PermissionsBoundary to empty string?
I thought I might be imagining the need for this line! I removed it and force-pushed.